api-docs-slate/source/includes/_clients.md

```diff
@@ -17,94 +17,99 @@ testnet   | 18332
 regtest   | 48332
 simnet    | 18556
 
-You can interact with bcoin with REST Api as well as RPC,
-there are couple of ways you can use API.
+You can interact with bcoin with its REST API as well as with RPC.
+There are couple of ways you can use the API:
 
-- `bcoin-cli` - has almost all methods described to be used.
-- `javascript` - Clients used by `bcoin-cli` can be used directly from javascript
-- `curl` - or you can use direct HTTP calls for invoking REST/RPC API calls.
+- `bcoin-cli` - methods built specifically into bcoin by its developers
+- `bcoin-cli rpc` - adds functionality that mimics Bitcoin Core RPC
+- `javascript` - methods used by `bcoin-cli` can be accessed directly from javascript
+- `curl` - you can use direct HTTP calls for invoking both REST and RPC API calls
 
-## Configuring BCOIN CLI
+## Configuring bcoin-cli
 
 ```shell--visible
-# You can use config file
```

Comment:
It should still be possible to use a conf file and will be looked for by bclient. We should also include some information about how the wallet will look for a separate conf file (this should be documented in the changelog)